tidb-cloud/data-service-manage-endpoint.md

```diff
@@ -233,11 +289,17 @@ TiDB Cloud generates code examples to help you call an endpoint. To get the code
     > **Note:**
     >
     > - By requesting the regional domain `<region>.data.tidbcloud.com`, you can directly access the endpoint in the region where the TiDB cluster is located.
-    > - Alternatively, you can also request the global domain `data.tidbcloud.com` without specifying a region. In this way, TiDB Cloud will internally redirect the request to the target region, but this might result in additional latency. If you choose this way, make sure to add the `--location-trusted` option to your curl command when calling an endpoint.
+    > - Alternatively, you can also request the global domain `data.tidbcloud.com` without specifying a region. In this way, TiDB Cloud Data Service will internally redirect the request to the target region, but this might result in additional latency. If you choose this way, make sure to add the `--location-trusted` option to your curl command when calling an endpoint.
 
-5. Paste the code example in your application and run it.
+5. Paste the code example in your application, edit the example according to your need, and then run it.
 
     - You need to replace the `<Public Key>` and `<Private Key>` placeholders with your API key. For more information, refer to [Manage an API key](/tidb-cloud/data-service-api-key.md).
+    - If the request method of your endpoint is `POST` or `PUT`, fill in the `--data-raw` option according to the rows of data that you want to operate.
+
+        - For endpoints with batch operations enabled, the `--data-raw` option accepts an array of data objects so you can operate multiple rows of data using one endpoint.
+        - For endpoints with batch operations not enabled, the `--data-raw` option only accepts one data object.
+
+    - If the request method of your endpoint is `DELETE` and the batch operation is enabled for the endpoint, you need to use `%2C` instead of comma (`,`) to separate the rows to be deleted, such as `/endpoint/<Endpoint Path>?id=${id}%2C${id}`. This is because comma (`,`) is a reserved character in URLs and cannot be used directly.
```

Comment:
```suggestion
        - For endpoints with **Batch Operation** enabled, the `--data-raw` option accepts an array of data objects so you can operate multiple rows of data using one endpoint.
        - For endpoints with **Batch Operation** not enabled, the `--data-raw` option only accepts one data object.

    - If the request method of your endpoint is `DELETE` and **Batch Operation** is enabled for the endpoint, you need to use `%2C` instead of comma (`,`) to separate the rows to be deleted, such as `/endpoint/<Endpoint Path>?id=${id}%2C${id}`. This is because comma (`,`) is a reserved character in URLs and cannot be used directly.
```